### PR TITLE
Merging in xbreaks/ydynam and other axis adjustments

### PR DESCRIPTION
**Pull request by pateldevenr**
I made adjustments to several scripts to include the dynamic y-axis calculations and change some x-axis intervals.

### CODE REVIEW

File: charts/code/agency_econ_significant_rules_by_presidential_year.Rmd
Comment (mfebrizio) [inline charts/code/agency_econ_significant_rules_by_presidential_year.Rmd:1]: File includes "More Testing" header. Is this finalized or testing still ongoing? If former I'd suggest removing the header before merging with main. If testing isn't done, let's resolve that first.

File: charts/output/cumulative_econ_significant_rules_by_first_year2.pdf
Comment (mfebrizio) [inline charts/output/cumulative_econ_significant_rules_by_first_year2.pdf:1]: please remove this file: we only want one cumulative first year chart in main

File: charts/output/cumulative_econ_significant_rules_by_first_year3.pdf
Comment (mfebrizio) [inline charts/output/cumulative_econ_significant_rules_by_first_year3.pdf:1]: please remove this file: we only want one cumulative first year chart in main

**Comment by pateldevenr** [comment]
This testing is finalized.  That was where I developed ydynam.  I have
removed it from my code and repushed.

File: charts/output/federal_register_pages_by_calendar_year_29.pdf
Comment (mfebrizio) [inline charts/output/federal_register_pages_by_calendar_year_29.pdf:1]: please remove this file and ensure changes have been made in the proper area: we only want one FR pages chart in main

File: charts/output/cumulative_econ_significant_rules_by_first_year2.pdf
Comment (mfebrizio) [inline charts/output/cumulative_econ_significant_rules_by_first_year2.pdf:1]: please remove this file: we only want one cumulative first year chart in main

File: charts/output/by_agency/cfr_pages_by_calendar_year.pdf
Comment (mfebrizio) [inline charts/output/by_agency/cfr_pages_by_calendar_year.pdf:1]: this is in the wrong folder

File: charts/code/agency_econ_significant_rules_by_presidential_year.Rmd
Comment (mfebrizio) [inline charts/code/agency_econ_significant_rules_by_presidential_year.Rmd:1]: File includes "More Testing" header. Is this finalized or testing still ongoing? If former I'd suggest removing the header before merging with main. If testing isn't done, let's resolve that first.

**Review by mfebrizio — CHANGES_REQUESTED** [review]

File: charts/output/cfr_pages_by_calendar_year.pdf
Comment (mfebrizio) [inline charts/output/cfr_pages_by_calendar_year.pdf:1]: let's ensure we have a version of this file in main.

File: charts/output/federal_register_pages_by_calendar_year.pdf
Comment (mfebrizio) [inline charts/output/federal_register_pages_by_calendar_year.pdf:1]: let's ensure we have a version of this file in main.

File: charts/output/by_agency/cfr_pages_by_calendar_year.pdf
Comment (mfebrizio) [reply charts/output/by_agency/cfr_pages_by_calendar_year.pdf:1]: Once this is deleted, we can merge the PR.

Comment (pateldevenr) [reply charts/output/by_agency/cfr_pages_by_calendar_year.pdf:1]: This was supposed to be archived, it has been removed.

File: charts/code/agency_econ_significant_rules_by_presidential_year.Rmd
Comment (pateldevenr) [reply charts/code/agency_econ_significant_rules_by_presidential_year.Rmd:1]: No longer needed, removed from code.

File: charts/output/federal_register_pages_by_calendar_year.pdf
Comment (pateldevenr) [reply charts/output/federal_register_pages_by_calendar_year.pdf:1]: Once merged, will output in main

File: charts/output/cfr_pages_by_calendar_year.pdf
Comment (pateldevenr) [reply charts/output/cfr_pages_by_calendar_year.pdf:1]: Once merged, will output in main

**Pull request merged**